### PR TITLE
#379 cart discount feature docs & release notes

### PR DESCRIPTION
**Pull request by ahmetoz**
cart discount feature docs & release notes

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -145,6 +156,7 @@ the library. `1.0.0` is here for you to use with all new features, enhancements
     - **Commons** - Bumped `org.junit.jupiter:junit-jupiter-engine` to 5.3.2.
     - **Commons** - Bumped `org.junit.jupiter:junit-jupiter-params` to 5.3.2.
     - **Commons** - Bumped `org.ajoberstar.git-publish` to 2.0.0.
+    - **Commons** - Bumped `com.adarshr.test-logger` to 1.6.0.
```
Comment (heshamMassoud) [inline docs/RELEASE_NOTES.md]: why is this added again? I see it already here: https://github.com/commercetools/commercetools-sync-java/pull/399/files#diff-aa75d2e7837774f5cb565717b6744a35R154

Comment (ahmetoz) [reply docs/RELEASE_NOTES.md]: aha, nice catch, a merge conflict issue 🙁
6d23157
